HLSPlugin/src/com/kaltura/hls/manifest/HLSManifestParser.as

```diff
@@ -87,11 +87,22 @@ package com.kaltura.hls.manifest
 
 		/**
 		 * After we have stalled once, we switch to using this as the minimum 
-		 * buffer period to allow playback.
+		 * buffer period to allow playback. Only used for VOD's
```

Comment:
I'd replace this with something like "Only used for VODs. Live streams have other logic (see LIVE_STREAM_BUFFER_THRESHOLD_MULTIPLIER)."